screenpy/protocols.py

```diff
@@ -60,18 +76,15 @@ def perform_as(self, the_actor: "Actor") -> None:
 
 
 @runtime_checkable
-class ErrorKeeper(Protocol):
-    """Classes that save exceptions for later are ErrorKeeper(s)"""
+class Resolvable(Protocol):
+    """Resolutions are Resolvable"""
 
-    caught_exception: Optional[Exception]
+    def resolve(self) -> Matcher:
+        """Form the Matcher for an assertion.
```

Comment:
Should resolve return BaseMatcher?

Reply:
No, because the matchers in PyHamcrest are `Matcher` objects (and `BaseMatcher`, the one external matchers are meant to inherit from, is also a `Matcher`). At least that's what i'm thinking.

If you look at the `is_equal` and all those functions from PyHamcrest, they all return some flavor of `Matcher`.